scripts/README.txt

```diff
@@ -0,0 +1,118 @@
+## Flow_stats's scripts
+
+This folder contains Flow_stats's related scripts.
+
+### Integration Kytos-ng and Zabbix for Monitoring
+
+[`kytos_zabbix.py`](./kytos_zabbix.py) is a script to help on the Kytos-ng monitoring for Zabbix.
+
+#### Pre-requisites
+
+- There's no additional Python libraries dependencies required (the libs we use are usually installed by default).
```

Comment:
I think it's great to ship related scripts as we've also done for `mef_eline`, `topology`, and `flow_manager`. Since this CLI tool also depends on `requests` that although ultimately will be available in the docker image, in the future, potentially shipping this as a python cli installable package could make things easier and having a `setup.py` installing its dependencies, that way it could also evolve as a tool that we support in its own repo one day and have also unit tests. No need to make changes here though, it's just a future suggestion.

Reply:
Excellent suggestion, Vinicius! Thanks! I will keep this in mind.